.editorconfig

```diff
@@ -112,6 +112,58 @@ dotnet_diagnostic.CS7035.severity = none
 # We do not need compile-time generated RegExs in Unit Tests. We'll prefer code readability.
 dotnet_diagnostic.SYSLIB1045.severity = none
 
+dotnet_diagnostic.MSTEST0001.severity = error
```

Comment:
Are these by default not already `error` ?

Reply:
@ynsehoornenborg most are not enabled by default.